### PR TITLE
Client Accounts Implementation

### PR DESCRIPTION
**Pull request by iCultureBud**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -66,18 +66,18 @@ func initServer() bool {
...
 
 		entities.ServerConfig.Server.Accounts[i].Nonce = id.String()
-		ServerUsers[user.UserName] = *user
+		ServerUsers[user.Name] = *user
 	}
 
 	tomlContent, err := toml.Marshal(entities.ServerConfig)
 	if err != nil {
-		zap.L().Error("Failed toml marshal")
+		zap.L().Error("Failed to parse config into struct.")
```
Comment (DrinoSan) [inline server/server.go]: We marshal the struct into a string.

File: entities/config.go [hunk 2]
```diff
@@ -15,9 +15,9 @@ type ServerConfig_t struct {
 }
 
 type ServerAccount_t struct {
-	UserName string `toml:"userName"`
+	Name string `toml:"userName"`
 	Nonce    string `toml:"nonce"`
```
Comment (DrinoSan) [inline entities/config.go:25]: UUID would now be better

**Review by DrinoSan — APPROVED** [review]

File: entities/config.go [hunk 3]
```diff
@@ -1,10 +1,12 @@
 package entities
 
-// Typedef for the yaml config object.
-type Config_t struct {
-	MotherShipIP   string `toml:"motherShipIP"`
-	MotherShipPort string `toml:"motherShipPort"`
-	MotherShipSSL  bool   `toml:"motherShipSSL"`
+// Typedef for the toml config objects.
+type ClientConfig_t struct {
+	IP    string `toml:"ip"`
```
Comment (DrinoSan) [inline entities/config.go:5]: Wo ist mothership?????

**Review by DrinoSan — APPROVED** [review]

File: entities/config.go (hunk 3 above)
Comment (iCultureBud) [reply entities/config.go:5]: It got crazy to write conditions on these laaarge keys 😩

**Pull request merged**